### PR TITLE
SDI-477: 🔥 Remove unit tests that are tested as functional tests

### PR DESCRIPTION
**Pull request by petergphillips**
- SDI-477: 🔥 Remove as tested in functional.pluginmanagers.SpringBootPluginManagerJarTest.Spring Boot info endpoint contains git info
- SDI-477: 🔥 Remove as tested in functional.configmanagers.BaseConfigManagerTest.Junit 5 tests can be executed
- SDI-477: 🔥 Remove as tested in functional.configmanagers.DependencyCheckPluginManagerTest
- SDI-477: 🔥 Remove as tested in functional.pluginmanagers.JavaSpringBootPluginManagerJarTest
- SDI-477: 🔥 Remove as tested in functional.pluginmanagers.KotlinPluginManagerTest

### CODE REVIEW

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/pluginmanagers/KotlinPluginManagerTest.kt [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-  fun `Should apply the Kotlin plugin`() {
-    project.plugins.getPlugin(KotlinPluginWrapper::class.java)
-  }
-
-  @Test
-  fun `Should apply Kotlin dependencies`() {
-    assertThat(project.configurations.getByName("implementation").dependencies)
-      .extracting<Tuple> { tuple(it.group, it.name) }
-      .contains(
-        tuple("org.jetbrains.kotlin", "kotlin-reflect")
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/pluginmanagers/KotlinPluginManagerTest.kt:22]: We're not actually testing these libraries are applied in the functional test. Should we?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/pluginmanagers/DependencyCheckPluginManagerTest.kt [hunk 2]
```diff
@@ -1,25 +0,0 @@
...
-
-  @Test
-  fun `Should apply the Owasp Dependency check plugin`() {
-    project.plugins.getPlugin(DependencyCheckPlugin::class.java)
-  }
-
-  @Test
-  fun `Should apply Owasp dependency check configuration`() {
-    val extension = project.extensions.getByName("dependencyCheck") as DependencyCheckExtension
-    Assertions.assertThat(extension.failBuildOnCVSS).isEqualTo(5f)
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/pluginmanagers/DependencyCheckPluginManagerTest.kt:20]: We're not testing that these configuration are applied in the functional test - should we?

**Comment by mikehalmamoj** [comment]
Also wondering if we should bump the version just in case we end up publishing by accident - would it override the existing version or not? (I don't actually know 😄 )

**Pull request merged**